### DISCUSSION TITLE
Empty Frame in Library and Playlist Tab on OLD iPad Air with iOS 12.5.7

### DISCUSSION THREAD

**Discussion by findnix** [rern/rAudio#1245] [Fixed Issues]
On my OLD, but still good iPad Air (iOS 12.5.7 / Safari and Chrome), both the main library frame and the playlist tab remain empty.

First of all: a big thank you for this excellent work!
I understand that the iOS and Webkit versions are very old, so it's not really a problem. But it was always very pleasant to control music with a tablet that is actually retired. And unfortunately, I can't tell if this behavior is due to an iOS update or an rAudio update.

Headers and buttons at the bottom work fine, a search in the Library tab shows the number of hits but no results. 
The same is true for the Playlist tab: The header correctly displays the number of items in the list, but the list remains empty. 
The Play tab works fine. 

I cannot observe this behavior on iPhones with iOS 16.3.1 (Safari, Firefox and Chrome) or Windows 11 with Chrome.

Greetings from Austria
Ronald

**Comment by rern** [comment]
Hi Ronald,

Fixed with today's update.

**Comment by findnix** [comment]
Well done! Works.

THANK YOU.
